Planes/ExperimentalPlane.js

```diff
@@ -1,4 +1,4 @@
-const Plane = require('./Plane');
```

Comment:
Why didn't you rename `Plane` folder in plane` as it was highlighted in the comments of the previous Pull Request?

Reply:
http://prntscr.com/225vyp8